lib/workers/branch/index.ts

```diff
@@ -393,6 +394,7 @@ export async function processBranch(
     removeMeta(['dep']);
 
     if (config.artifactErrors?.length) {
+      await setArtifactErrorStatus(config);
```

Comment:
Are you sure that a branch/commit exists at this point? there's a commit files call on line 444 after it

Reply:
🤔 will check it again